### PR TITLE
Add the path_raw field to a request object

### LINKED ISSUE #56: Request path is decoded before the arguments are splitted
Route:
`httpd:route({ path = '/path/*data' }, 'controller#method' )`
URI:
`domain.com/path/first=second%2Fthird/fourth (where %2F is encoded slash "/")`
Controller:
I can't split by real slash because the encoded symbol is decoded.

Can I get the encoded request path?

### PR DESCRIPTION
**Pull request by lowitea**
Add a ability to get the request path without decoding

Maybe it closes #56, but I'm not sure :)

### CODE REVIEW

**Comment by lowitea** [comment]
@Totktonada Hi!
Sorry to bother you
But could someone look at the PR?)

**Comment by Totktonada** [comment]
@lowitea Sorry for the delay. I marked this PR to review and hopefully will look on this/next week.

**Comment by lowitea** [comment]
@Totktonada ok :) Thank you.

**Review by 0x501D — APPROVED** [review]

**Comment by lowitea** [comment]
@Totktonada Hi
I'm sorry to bother you again, but I would really like to merge this pr :) If it is possible of course

**Review by Totktonada — APPROVED** [review]
Sorry again for the delay.

That's really strange that `req.path` doesn't allow durable splitting between path parts. Maybe we have some design flaw here.

Anyway, I don't mind the change per se. @0x501D will proceed.

**Comment by lowitea** [comment]
@Totktonada 
Thank you!
Could you release a new version with this change after merge the pr?

**Comment by 0x501D** [comment]
> @Totktonada Thank you! Could you release a new version with this change after merge the pr?

I think it can be done.

**Pull request merged**
